main/src/main/scala/sbt/internal/parser/SbtParser.scala

```diff
@@ -99,14 +98,15 @@ private[sbt] object SbtParser {
   }
 
   private[sbt] final val globalReporter = new UniqueParserReporter
+  private[sbt] var scalacGlobalInitReporter: Option[ConsoleReporter] = None
 
   private[sbt] final lazy val defaultGlobalForParser = {
-    import scala.reflect.internal.util.NoPosition
     val options = "-cp" :: s"$defaultClasspath" :: "-Yrangepos" :: Nil
-    val reportError = (msg: String) => globalReporter.error(NoPosition, msg)
+    val reportError = (msg: String) => System.err.println(msg)
```

Comment:
No, not at all. In fact, this should never happen. It happened only in this case because I was fiddling with the classpath of the compiler. But this should not affect to normal users.